libcaf_core/caf/actor_system_config.hpp

```diff
@@ -252,6 +252,13 @@ class actor_system_config {
   size_t work_stealing_relaxed_steal_interval;
   size_t work_stealing_relaxed_sleep_duration_us;
 
+  // -- config parameters for the logger ---------------------------------------
+
+  std::string logger_filename;
+  std::string logger_verbosity;
```

Comment:
Nevermind, turns out some compiler don't think `constexpr` functions qualify for constant expressions. 😒

Here's a workaround with `atom_constants`: https://github.com/actor-framework/actor-framework/blob/master/libcaf_io/src/basp_broker.cpp#L239